elfo-network/src/worker/flows_rx.rs

```diff
@@ -41,11 +41,24 @@ impl Drop for RxFlows {
     }
 }
 
+#[derive(Debug)]
+pub(super) enum RxFlowEvent {
```

Comment:
I would prefer avoid using "Event" because events are messages (on a par with commands).

The "Message" variant is also confusing because "Response" is also message. Do you mean regular and requests here, right?